tt_metal/impl/device/device.hpp

```diff
@@ -389,6 +389,11 @@ class Device {
     void clear_loaded_sub_device_manager();
     void remove_sub_device_manager(SubDeviceManagerId sub_device_manager_id);
     const std::vector<SubDeviceId> &get_sub_device_ids() const;
+
+    // Temporary api until migration to actual fabric is complete
```

Comment:
Can you add a TODO with the issue number here? So that it shows up on github in the issues directly.

Reply:
Done